src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -0,0 +1,144 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.2;netstandard1.3;net45;win81;wpa81;MonoAndroid70;Xamarin.iOS10</TargetFrameworks>
```

Comment:
Do you plan on supporting `win81` in this release? If so, `wpa81` is identical; no changes at all.